IFC4x3/PropertyConstants/y/YESCLOTHOID_1mbTFnm$v5VOyBNm1BiBGi.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="YESCLOTHOID_1mbTFnmv5VOyBNm1BiBGi" Name="YESCLOTHOID" UniqueId="7095d3f1-c3fe-457d-8f0b-5f004bb0b42c" />
```

Comment:
Can the type of the transition not be extracted from the alignment?

Reply:
WIP -> Question To Claude, Andreas and domain expert.

Reply:
Straight / CircularArc / Transition ?

Reply:
Done. Commit : e5e747f8ce6e67ed1e5301b5fb1fba218df02d67